stage1/usr_from_src/libnss.mk

```diff
@@ -0,0 +1,18 @@
+$(call setup-stamp-file,UFSN_STAMP)
+LIBNSS_FILES_LINK := libnss_files.so.2
```

Comment:
Also, could you add a TODO here about moving this kind of functionality into find-so-deps.mk? That `ld --verbose | gobbledygook` trick is what I need there.